zephyr/CMakeLists.txt

```diff
@@ -88,6 +88,10 @@ target_include_directories(SOF INTERFACE ../zephyr/include)
 target_include_directories(SOF INTERFACE ${SOF_SRC_PATH}/include)
 target_include_directories(SOF INTERFACE ${SOF_SRC_PATH}/arch/${ARCH}/include)
 
+# TODO: The xtos-wrapper include PATH will eventually become optional based on the
+# RTOS selection. It's manadatory atm until the native Zephyr integration is ready.
+target_include_directories(SOF INTERFACE ${SOF_SRC_PATH}/arch/xtos-wrapper/include)
```

Comment:
I'm still not sure about this, I'd rather not do this but maybe move headers more gradually - only those that we don't need with Zephyr already now. Besides doesn't this create a confusion for headers like zephyr/spinlock.h? Now there's one in Zephyr itself and one in xtos-wrapper...

Reply:
This will be moving header usage gradually, but we need to put all the RTOS/IP headers in a place today where this can be visualized as today they are scattered around (making it difficult to debug issues where contamination is suspected).